app/controllers/spam_reports_controller.rb

```diff
@@ -2,8 +2,8 @@ class SpamReportsController < ApplicationController
   wrap_parameters false
 
   def create
-    delivery_attempt = DeliveryAttempt.find(params[:reference])
-    SpamReportService.call(delivery_attempt)
+    subscriber = Subscriber.find_by(address: params[:to])
```

Comment:
Sorry I missed this before, this should use [the find_by_address method](https://github.com/alphagov/email-alert-api/blob/57c1abf9d512331d674cba60055b8e23c4c80c3f/app/models/subscriber.rb#L25)